### PR TITLE
Technical author review of maestro_config.md

### PR DESCRIPTION
**Pull request by GuyWi**
@psppalash,  @mray190 can you please review? Sorry, I did not have permission to write to armPelionEdge/maestro.

### CODE REVIEW

File: docs/maestro_config.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ network:
...
+* `if_name` - REQUIRED. Name of the interface Maestro modifies.
+* `existing` - OPTIONAL. Tells Maestro to `replace` or `override` the existing saved interface.
+* `clear_addresses` - REQUIRED. Clears any existing addresses assigned to the interface before setting up the specified addresses.
+* `dhcpv4` - REQUIRED. `false` for static interfaces.
+* `ipv4_addr` - REQUIRED. IP address to assign to the interface.
+* `ipv4_mask` - REQUIRED. IP mask to use for the subnet.
+* `hw_addr` - MAC address to use for the interface.
 
-### DHCP interface:
+### DHCP interface
```
Comment (GuyWi) [inline docs/maestro_config.md:46]: The examples for static interface and DHCP interface seem to be identical. Is this correct?

Comment (psppalash) [reply docs/maestro_config.md:46]: Thank You for pointing this out. It is not correct. I did notice this a few days back and have corrected it. My changes are already in dev branch which will be merged into master in the next release.

**Comment by psppalash** [comment]
All the changes look good to me.

**Comment by GuyWi** [comment]
Thanks, @psppalash! 
Can you please merge this into armPelionEdge/maestro?

**Comment by psppalash** [comment]
@mray190 Should we merge this PR to dev or master?

**Comment by mray190** [comment]
These changes are against master, which is production. We have commits on `dev` that have updated some of the things that are mentioned in this PR. Can you please rebase your commits on `dev` and fix up any conflicts?
Or @psppalash if you have extra bandwidth can you help with this?

**Comment by GuyWi** [comment]
@psppalash, @mray190 I rebased on `dev` (or at least tried... sorry if I made a bit of a mess, but I'm pretty sure maestro_config.md is now aligned with `dev` :) ).

**Comment by psppalash** [comment]
Thank You @GuyWi . I will take a look.

**Pull request merged**